src/pages/Transaction/TransactionCellScript/index.tsx

```diff
@@ -333,8 +333,24 @@ export default ({ cell, onClose, txStatus }: { cell: State.Cell; onClose: Functi
           <div className="transaction__detail_copy">
             <TransactionDetailCopyButton onClick={onClickCopy}>
               <div>{i18n.t('common.copy')}</div>
-              <img src={isMainnet() ? CopyIcon : CopyBlueIcon} alt="copy" />
+              <CopyIcon />
             </TransactionDetailCopyButton>
+            {(state === CellState.LOCK || state === CellState.TYPE) &&
+              content &&
+              (content as State.Script).codeHash &&
+              (content as State.Script).hashType && (
```

Comment:
You shouldn't use `as` to assert, you should use automatic inference, like `content && 'codeHash' in content`.

The type of `content` in this conditional branch is inferred to be `State.Script`, and `as` is not required for use below.

Reply:
> You shouldn't use `as` to assert, you should use automatic inference, like `content && 'codeHash' in content`.
> 
> The type of `content` in this conditional branch is inferred to be `State.Script`, and `as` is not required for use below.

fixed